homeassistant/helpers/entity_platform.py

```diff
@@ -291,7 +296,24 @@ async def async_add_entities(
         if not tasks:
             return
 
-        await asyncio.gather(*tasks)
+        await asyncio.wait(tasks, timeout=SLOW_ADD_ENTITIES_MAX_WAIT)
+
+        for idx, entity in enumerate(new_entities):
```

Comment:
We have a pending task logger function helper on the core object, maybe we should extend this for custom logging messages?

Reply:
I didn't specifically log the tasks here since it will also be the _async_add_entity function which didn't seem useful.